server/validators/allocation.js

```diff
@@ -0,0 +1,36 @@
+const yup = require('yup');
+const { validateBlankOrPositiveInteger } = require('./helpers');
+
+const CreateAllocationSchema = yup
+  .object()
+  .noUnknown('Unknown field in entry')
+  .shape({
+    allocation: yup
+      .number()
+      .nullable()
+      .max(99, 'Must be between 0-99')
```

Comment:
Ah shoot sorry I meant to say, no max limits!
Please remove the related test too!

Reply:
removed!